src/microbe_stage/Microbe.cs

```diff
@@ -1023,6 +1023,9 @@ private Node GetStageAsParent()
         if (Colony == null)
             return GetParent();
 
+        if (Colony.Master.HostileEngulfer.Value != null)
```

Comment:
When is this not null, doesn't the fact that when the engulfing starts the cell is removed from the colony mean that if the master is engulfed, it will just immediately break the colony? Or is this needed in this case. Could you clarify this with a comment in the source code?